admin/index.php

```diff
@@ -0,0 +1,28 @@
+<?php
+include('../include/initialize.php');  
+echoHeader('Students');
+
+echo"
+        <h1>Students List</h1>    
+    ";
+
+$allStudents = getAllStudents();
+
+foreach($allStudents as $student){
+    echo"
+    <div>
+        <b>Student ID: </b>" .$student['Student_Id']."</br> 
+        <b>First Name: </b> " .$student['First_Name']."</br> 
+        <b>Last Name: </b>" .$student['Last_Name']."</br> 
+        <b>Email: </b>" .$student['Email']."</br> 
+        <b>Phone: </b>" .$student['Phone']."</br> 
+        <b>Date Created: </b>" .formatDate($student['Date_Created'], "m/d/y")."</br> 
+        <b>Private Notes: </b>" .$student['Private_Notes']."</br>";                
```

Comment:
I mentioned this to him in person, and he said this is something you all will cover later in the summer. So I wouldn't worry about it for now, since it sounds like you'll be working on it later.